velox/functions/sparksql/Register.cpp

```diff
@@ -97,18 +97,12 @@ void registerFunctions(const std::string& prefix) {
       prefix + "rlike", re2SearchSignatures(), makeRLike);
   VELOX_REGISTER_VECTOR_FUNCTION(udf_regexp_split, prefix + "split");
 
-  // Subscript operators. See ExtractValue in complexTypeExtractors.scala.
```

Comment:
Let's mention these changes in the PR description. Or, better yet, extract these into a separate PR.

Reply:
I extracted these into https://github.com/facebookincubator/velox/pull/4095.